{"name": "Wrong data authorization", "status": "passed", "steps": [{"name": "open the login page...", "status": "passed", "steps": [{"name": "open the page", "status": "passed", "start": 1715785047680, "stop": 1715785047999}], "start": 1715785047680, "stop": 1715785047999}, {"name": "log in with the wrong username and password...", "status": "passed", "steps": [{"name": "open the page", "status": "passed", "start": 1715785048000, "stop": 1715785048010}, {"name": "fill in the fields...", "status": "passed", "start": 1715785048010, "stop": 1715785048092}, {"name": "click the login button...", "status": "passed", "start": 1715785048093, "stop": 1715785048131}], "start": 1715785047999, "stop": 1715785048131}, {"name": "check the current url and the error message is provided...", "status": "passed", "steps": [{"name": "get the current page url", "status": "passed", "start": 1715785048132, "stop": 1715785048136}], "start": 1715785048131, "stop": 1715785048136}, {"name": "take full screenshot in headless mode...", "status": "passed", "start": 1715785048136, "stop": 1715785048247}], "start": 1715785047680, "stop": 1715785048247, "uuid": "0c429c3f-cd4d-484d-a73a-802ff2f6a655", "historyId": "0812e83d1e44db295595ba7d0ba5f53e", "testCaseId": "0812e83d1e44db295595ba7d0ba5f53e", "fullName": "tests.test_auth.TestLogin#test_wrong_login_auth", "labels": [{"name": "epic", "value": "Auth page"}, {"name": "as_id", "value": "1.5"}, {"name": "feature", "value": "Auth"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "POLA-AI"}, {"name": "thread", "value": "18720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}